dist/focus-ring.js

```diff
@@ -253,7 +200,19 @@ document.addEventListener('DOMContentLoaded', function() {
    * @return {boolean}
    */
   function focusTriggersKeyboardModality(el) {
-    return index$1(el, keyboardModalityWhitelist) && index$1(el, ':not([readonly])');
+    var type = el.type;
+    var tagName = el.tagName.toLowerCase();
+
+    if (tagName == 'input' && inputTypesWhitelist[type] && !el.readonly)
+      return true;
+
+    if (tagName == 'textarea' && !el.readonly)
+      return true;
+
+    if (el.contentEditable)
```

Comment:
The possible values of `contentEditable` are `'inherit'`, `'true'`, or `'false'`.